src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -146,8 +146,16 @@ export default {
 	},
 	metaInfo() {
 		const canonicalUrl = `https://${this.$appConfig.host}${this.$route.path}`.replace('-beta', '');
+		let title = '';
+		let description = '';
+
+		if (this.$route.path.includes('lend') || this.$route.path.includes('-beta')) {
+			title = `Lend to ${this.name} in ${this.countryName}`;
```

Comment:
We need to consider the loans anonymization level. If it is full then we should hide the name and loan description like it is done in the computed properties.